tests/test_bench.py

```diff
@@ -763,3 +763,22 @@ def test_oqc_postprocessing():
     )
     assert QuantumCircuit.from_qasm_file(str(path))
     path.unlink()
+
+
+NUM_EVAL_ATTRIBUTES = 10
+
+
+def test_evaluate_qasm_file():
+    qc = get_benchmark("dj", 1, 5)
+    filename = "test_5.qasm"
+    qc.qasm(filename=filename)
+    path = Path(filename)
+    res = evaluation.evaluate_qasm_file(filename)
+    assert len(res) == NUM_EVAL_ATTRIBUTES
+    assert res[0] == filename
+    for elem in res[1:5]:
+        assert isinstance(elem, int)
+    for elem in res[5:]:
+        assert isinstance(elem, float)
```

Comment:
I'd say most of these assertions become redundant when the evaluation returns a TypedDict.
These tests could be a little more comprehensive I guess.